redaxo/src/core/default.config.yml

```diff
@@ -10,6 +10,12 @@ fileperm: '0664'
 dirperm: '0775'
 session_duration: 7200
 session_keep_alive: 21600
+backend_login_policy:
+    login_tries_1: 3
+    relogin_delay_1: 5
+    login_tries_2: 50
+    relogin_delay_2: 3600
+    enable_stay_logged_in: true
```

Comment:
Die Keys müssen in die Schema-Datei aufgenommen werden: https://github.com/redaxo/redaxo/blob/main/redaxo/src/core/schemas/config.json

Reply:
interessant.. kommt bei mir kein error... muss man die json validierung explizit irgendwo aktiviere?

Reply:
hier im repo ist sie eigentlich automatisch hinterlegt/aktiviert.
in redaxo-projekten bisher leider nicht, da muss man es selbst konfigurieren. aber die datei wird mit ausgeliefert.

ich bin gerade nicht sicher, ob man noch irgendein phpstorm-plugin benötigt.

Reply:
<img width="618" alt="Bildschirmfoto 2022-07-14 um 15 31 51" src="https://user-images.githubusercontent.com/330436/178994432-c61b872f-5450-4fcb-8a40-741b225e7725.png">

<img width="519" alt="Bildschirmfoto 2022-07-14 um 15 32 15" src="https://user-images.githubusercontent.com/330436/178994448-682caf90-4d6a-4a9b-9e74-6aeade485d55.png">

Reply:
ist bei mir nicht so schick, aber geht auch:

<img width="648" alt="grafik" src="https://user-images.githubusercontent.com/120441/178995136-7c0dbc86-255c-4478-abff-65de0c396748.png">


ggf. hast du EAP? interessant, dass wir `use_https` mit leerstring beschreiben - im schema aber nicht erlauben

Reply:
Aktuell habe ich nicht die EAP, sondern letzte Stable.